packages/js/product-editor/src/components/iframe-editor/editor-canvas.tsx

```diff
@@ -0,0 +1,63 @@
+/**
+ * External dependencies
+ */
+import { createElement, Fragment } from '@wordpress/element';
+import {
+	// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+	// @ts-ignore No types for this exist yet.
+	__unstableEditorStyles as EditorStyles,
+	// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+	// @ts-ignore No types for this exist yet.
+	__unstableIframe as Iframe,
+	// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+	// @ts-ignore No types for this exist yet.
+	__unstableUseMouseMoveTypingReset as useMouseMoveTypingReset,
+} from '@wordpress/block-editor';
+
+type EditorCanvasProps = {
+	enableResizing: boolean;
+	children: React.ReactNode;
+};
+
+export function EditorCanvas( {
+	enableResizing,
+	children,
+	...props
+}: EditorCanvasProps ) {
+	const mouseMoveTypingRef = useMouseMoveTypingReset();
+	return (
+		<Iframe
+			head={
+				<>
+					<EditorStyles />
```

Comment:
It seems like this takes a `styles` prop, or relies on one for me. It errors and blank screens without providing one.
Maybe this has been recently added?

Reply:
I did not manage to hit the error, but since we're not really passing any extra styles, I think we can remove it until we want to pass settings down.